docs/PR_XXX_OPENING_CHECKLIST.md

```
@@ -0,0 +1,156 @@
+- Technical debt report: docs/audit/PR_XXX_TECHNICAL_DEBT_REPORT.md
+
+See: docs/audit/PR_XXX_THIN_CLIENT_HTTP_ADAPTER_AUDIT_TEMPLATE.md"
+
+# Push branch
+git push -u origin feat/ios-thin-http-adapter
+```
+
+### 2. Create PR on GitHub
+
+**Title:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)